### ISSUE TITLE
Bug: Docs - Activation function used is relu not sigmoid

### ISSUE THREAD

**Issue by fawazahmed0** [tensorflow/hub#877]
### What happened?

You should update [this](https://www.tensorflow.org/hub/tutorials/tf2_text_classification#:~:text=layer%20with%20a-,sigmoid%20activation,-\)%2C%20we%27ll%20use%20the) , the activation function used in the example is relu

### Relevant code

```python
NA
```

### Relevant log output

```shell
NA
```

### tensorflow_hub Version

0.13.0.dev (unstable development build)

### TensorFlow Version

2.8 (latest stable release)

### Python Version

3.x

### OS

Linux

**Comment by singhniraj08** [comment]
@fawazahmed0,

`Relu` activation is used in the layer with 16 hidden units. `Sigmoid` activation is referenced for the last output layer with single output node. We can also write the last output layer as 
```
model.add(tf.keras.layers.Dense(1, activation='sigmoid'))
```

Since, we have mentioned `from_logits=True` in loss `loss=tf.keras.losses.BinaryCrossentropy(from_logits=True)` in the model compile, we have not mentioned the `sigmoid` activation in last model layer. The model outputs single floating-point value which either represents a [logit](https://en.wikipedia.org/wiki/Logit), (i.e, value in [-inf, inf] when from_logits=True).

Thank you!

**Comment by fawazahmed0** [comment]
Thanks
